smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ServiceClientJSONGenerator.kt

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Suggestion on naming: something like `DependencyJSONGenerator` is probably more clearer than `ServiceClientJSONGenerator`. Relevant to all places where this name is used.